.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-unknown-linux-gnu]
+linker = "x86_64-linux-gnu-gcc"
```

Comment:
I don't get it. You're specifying linker for x86 target, how is that relevant to arm cross built? :thinking:

Reply:
x86 is the target for the cross compilation.

Reply:
ah ok. Thing is that this will be used on normal native builds too and that would not be good. E.g I use mold linker for faster linking and AFAICT, this would override that.